core/src/main/java/org/streamprocessor/core/transforms/SerializeMessageToRowFn.java

```diff
@@ -150,14 +155,27 @@ public SerializeMessageToRowFn(
         this.ratio = 0.001f;
     }
 
-    @Setup
-    public void setup() throws Exception {}
+    public static TableRow convertJsonToTableRow(String json) {
```

Comment:
is this the same as the jsontotablerow.java in utils folder? Maybe use that instead instead?